src/routers/v1/RouteRouter.js

```diff
@@ -40,7 +40,8 @@ class RouteRouter extends ApplicationRouter<Array<Object>> {
         time,
         uid,
       };
-      LogUtils.log({ category: 'routeRequest', request });
+      const containsTransfer = routes.find(route => RouteUtils.routeContainsTransfer(route)) !== undefined;
+      LogUtils.log({ category: 'routeRequest', request, containsTransfer });
```

Comment:
we should only log if it actually contains a transfer, not if it doesn't (the majority of routes don't contain transfers)

Reply:
well right now we log every request anyways so I just added a field to that log

Reply:
discussed offline, we should remove this log